server/plugin/virtual_agent_test.go

```diff
@@ -239,15 +240,15 @@ func Test_CreateOutputImagePost(t *testing.T) {
 			description: "Image post is created",
 			body: &OutputImage{
 				Value:   "https://test/test.jpg",
-				AltText: "mockAltText",
+				AltText: mockAltText,
```

Comment:
I didn't mean this "AltText", I meant the whole body is same in every test case.